src/test/java/org/htmlunit/javascript/host/LocationTest.java

```diff
@@ -412,30 +415,30 @@ public void reloadWithHash() throws Exception {
         String date = page.getElementById("date").asNormalizedText();
         page = page.getElementById("reload").click();
         String newDate = page.getElementById("date").asNormalizedText();
-        assertEquals(date, newDate); // FIXME: this should be assertNotEquals
+        assertNotEquals(date, newDate);
 
         Thread.sleep(100);
 
         date = newDate;
         page = page.getElementById("updateHashThenReload").click();
         newDate = page.getElementById("date").asNormalizedText();
-        assertEquals(date, newDate); // FIXME: this should be assertNotEquals
+        assertNotEquals(date, newDate);
 
         Thread.sleep(100);
 
         date = newDate;
         page = page.getElementById("updateHashThenReload").click();
         newDate = page.getElementById("date").asNormalizedText();
-        assertEquals(date, newDate); // FIXME: this should be assertNotEquals
+        assertNotEquals(date, newDate);
 
         String[] expected = {
                 "hash: #0",
                 "reload",
-                // FIXME: missing "hash: #0"
+                "hash: #0",
                 "update hash then reload",
-                // FIXME: missing "hash: #1"
+                "hash: #0",
```

Comment:
// FIXME: this should be "hash: #1"

Reply:
Thank you for pointing that out! I've made the corrections with this commit🙇‍♂. [d76e9a9](https://github.com/yamamoto-kanoko/htmlunit/pull/1/commits/d76e9a9aa2562ea4db4bbc132125d5525b7949a0)